### PR TITLE
Extract Indicators From File - Generic v2 - fixed error with XLSB

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
- [x] In Progress

## Description
Fixed an issue where the presence of password protected / XLSB files caused an error when converting them.

Tested with both XLSX and XLSB in the context to ensure one is not caught and the other is:
image

### CODE REVIEW

**Review by tomer-pan — APPROVED** [review]
Approved.
Maybe it's worth adding in the playbook description that it's not handling such files at the moment.

File: Packs/CommonPlaybooks/ReleaseNotes/2_6_39.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Extract Indicators From File - Generic v2
+
+- Fixed an issue where the file conversion tasks attempted to convert password-protected XLSB files which would result in an error.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_6_39.md:6]:
```suggestion
Fixed an issue where the file conversion tasks attempted to convert password-protected XLSB files which resulted in an error. 
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret @tomer-pan @altmannyarden Doc review completed.

**Comment by idovandijk** [comment]
Will merge this on Sunday as a precaution for a sensitive playbook.

**Pull request merged**